src/phpDebug.ts

```diff
@@ -279,10 +279,6 @@ class PhpDebugSession extends vscode.DebugSession {
                         })
                         connection.on('error', disposeConnection)
                         connection.on('close', disposeConnection)
-                        connection.on('before-execute-command', () => {
```

Comment:
Is this event then still used at all?

Reply:
@felixfbecker No, this PR completely removes the event, it turns out that it causes problems to send the continued event.  In several scenarios it duplicates other events sent that also tell vscode that the code is running.  The second notice sometimes gets sent after it is paused again, and ends up making vsc think it continued when it is actually still paused.